resources/views/dielo.blade.php

```diff
@@ -324,6 +324,12 @@ class="no-border license" title="Public Domain" data-toggle="tooltip"><img
                     @endif
 
                     <div class="col-md-12 text-center">
+                        <user-collections-favourite-button
+                            label-add="{{ utrans('general.item_add_to_favourites') }}"
+                            label-remove="{{ utrans('general.item_remove_from_favourites') }}"
+                            id="{{ $item->id }}"
+                            is-detail=true
```

Comment:
```suggestion
                            is-detail
```
would've also worked